### DISCUSSION TITLE
Feature Request: Integration with Immich (photos on the map)

### DISCUSSION THREAD

**Discussion by aviv926** [Freika/dawarich#225] [Ideas]
This looks like a great project, and looks like it could work great with Immich in terms of pulling photos taken along today's timeline (as [Google does a timeline integration with Google Photos](https://www.gadgets360.com/apps/news/google-photos-maps-timeline-update-adds-new-feature-picture-routes-2339967))
immich has an [API interface](https://immich.app/docs/api) that I think with the right combination it could work just great and then it would be the excellent alternative :)

**Comment by Freika** [comment]
I think at some point I'll introduce something similar to what you're suggesting, but for now I'm more focused on making the app more stable. Thanks for the idea!

**Comment by Freika** [comment]
It's done: https://github.com/Freika/dawarich/releases/tag/0.17.0
